packages/console/src/ConsoleMenu.tsx

```diff
@@ -182,9 +183,9 @@ class ConsoleMenu extends PureComponent<ConsoleMenuProps, ConsoleMenuState> {
 
     return (
       <div className="console-pane-menu">
-        <button
-          type="button"
-          className="btn btn-link btn-link-icon"
+        <Button
+          kind="ghost"
```

Comment:
Add dummy `onClick` handlers here to get rid of the proptype warnings (as per the other review)